### PR TITLE
fix: `NameError: name 'url' is not defined` for invalid bb url

### PR DESCRIPTION
**Pull request by benoror**

### CODE REVIEW

**Comment by benoror** [comment]
Originally reported: http://webapps.stackexchange.com/questions/49729/how-can-i-import-issues-from-bitbucket-to-github#comment76242_49900

File: migrate.py [hunk 1]
```diff
@@ -283,7 +284,7 @@ def get_issues(bb_url, start):
             raise RuntimeError(
                 "Could not find the Bitbucket repository: {url}\n"
                 "Hint: the Bitbucket repository name is case-sensitive."
-                .format(url=url)
+                .format(url=bb_url)
```
Comment (jeffwidman) [inline migrate.py]: This is indeed a typo that needs fixing

File: migrate.py [hunk 2]
```diff
@@ -94,7 +94,8 @@ def main(options):
     """
     Main entry point for the script.
     """
-    bb_url = "https://api.bitbucket.org/1.0/repositories/{repo}/issues".format(
+    bb_url = "https://api.bitbucket.org/1.0/repositories/{account}/{repo}/issues".format(
+        account=options.bitbucket_username,
```
Comment (jeffwidman) [inline migrate.py]: This is incorrect. Reason being that I'll be adding support for private repos, and sometimes the bb username of a collaborator might be different from the repo's owner username.

File: migrate.py [hunk 3]
```diff
@@ -295,7 +296,7 @@ def get_issues(bb_url, start):
         else:
             raise RuntimeError(
                 "Bitbucket returned an unexpected HTTP status code: {code}"
-                .format(respo.status_code)
+                .format(code=respo.status_code)
```
Comment (jeffwidman) [inline migrate.py]: :+1: And another typo that needs fixing.

**Comment by jeffwidman** [comment]
If you update your PR to be a single commit fixing the two typos (and dropping the incorrect change), then I'll happily merge it.

**Comment by benoror** [comment]
Feel free to fix the typos in a separate PR.

Cheers!

**Pull request closed without merging**